src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -77,6 +77,8 @@ namespace PhotovoltaicThermalCollectors {
         FIXED
```

Comment:
`enum` that will be read from the IDF need to start with `Invalid = -1` and end with `Num`.

Reply:
updated